executor/show.go

```diff
@@ -1163,3 +1167,119 @@ func (e *ShowExec) appendRow(row []interface{}) {
 		}
 	}
 }
+
+func (e *ShowExec) fetchShowTableRegions() error {
+	store := e.ctx.GetStore()
+	tikvStore, ok := store.(tikv.Storage)
+	if !ok {
+		return nil
+	}
+	splitStore, ok := store.(splitableStore)
+	if !ok {
+		return nil
+	}
+
+	tb, err := e.getTable()
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	// Get table regions from from region cache.
+	regionCache := tikvStore.GetRegionCache()
```

Comment:
Get region from PD directly.

Reply:
Currently is get region from pd directly. But I need to call the related `regionCache` method to get region from pd.

Reply:
So please update the comment in line 1187

Reply:
done.